src/components/organisms/cards/cards.scss

```
@@ -0,0 +1,31 @@
+@use "../../variables" as *;
+
+.organisms-cards {
+  .container {
+    max-width: 100%;
+    padding-right: 0;
+    padding-left: 0;
+    @media (max-width: $breakpoint-md) {
+      margin-right: auto;
+      margin-left: auto;
+      max-width: 540px;
+    }
+  }
+  .cards{
+    display: flex;
+    justify-content: space-around;
```

```suggestion
    justify-content: space-between;
```
space-aroundは左右端にも余白が発生してしまいます。
今回は左右端の余白は親要素 `.container` の方で設定しているので、こちらでは `space-between`  を使って子要素間の余白だけを設定するのが望ましいです

---

対応しました

7447352298afd2ee693fdef833d126ae0ec4f6c9